src/capitalsFirst.js

```diff
@@ -21,7 +21,18 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let sortWords = [];
```

Comment:
you dont need let if you dont reassign a variable [prefer-const rule](https://mate-academy.github.io/style-guides/javascript.html#variables--const)